tests/integration/relationships/inverse-relationships-test.js

```diff
@@ -463,8 +463,8 @@ testInDebug("Inverse relationships that don't exist throw a nice error for a bel
   }, /We found no inverse relationships by the name of 'testPost' on the 'user' model/);
 });
 
-test("inverseFor is only called when inverse is not null", function(assert) {
-  assert.expect(2);
+test("inverseFor short-circuits when inverse is null", function(assert) {
```

Comment:
Now that we have both `inverseFor` and `_inverseFor` should we test both of them?